react/Button/Button.less

```diff
@@ -111,3 +111,47 @@
 .fullWidth {
   width: 100%;
 }
+
+.group {
+  display: flex;
+  width: 100%;
+
+  .root {
+    overflow: visible;
+
+    & + .root {
+      margin-left: @grid-gutter-width / 2;
+    }
+
+    & + .root_isTransparent {
+      margin-left: 0;
+    }
+  }
+
+  .root_isTransparent {
+    padding-left: @grid-gutter-width;
+    padding-right: @grid-gutter-width;
+  }
+  @media only screen and (max-width: 960px) {
+    .root {
+      flex: 33.33% 0;
```

Comment:
I'd think perhaps it would make sense to pass from outside in each individual case

Reply:
Interested to know why the 33.3%? Wouldnt this be better expressed as a `max-width`?